### PR TITLE
[TW-1623] Add new GraphQL Schema tab features

### PR DESCRIPTION
**Pull request by petewitcher-postman**
Updates one paragraph.

### CODE REVIEW

**Review by mark-anthony-postman — APPROVED** [review]
LGTM!

**Review by jkonrath-postman — APPROVED** [review]
LGTM

File: src/pages/docs/sending-requests/graphql/graphql-client-interface.md [hunk 1]
```diff
@@ -72,7 +72,7 @@ In the GraphQL client's request section, you can enter an endpoint in the URL fi
     * **Query** - The **Query** tab has three subsections: the schema explorer, the query editor, and the variables editor.
     * **Authorization** - You can enter authorization credentials here. You can choose from a list of auth types including API key, basic auth, and bearer token. Learn more about [authorizing requests](/docs/sending-requests/authorization/authorization).
     * **Headers** - The **Headers** tab enables you to pass headers with the request in the form of key-value pairs. The client uses these headers to provide information about the call to the server.
-    * **Schema** - The **Schema** tab enables you to select an API from a Postman workspace, import a schema from your computer, or use GraphQL introspection. The GraphQL client populates the schema explorer with fields from the selected schema.
+    * **Schema** - The **Schema** tab enables you to select an API and save it to the request. You can select an API from a Postman workspace, or import a schema from a local file or a URL. You can also select a specific published release of an API. The GraphQL client populates the schema explorer with fields from the chosen schema.
```
Comment (ronork) [inline src/pages/docs/sending-requests/graphql/graphql-client-interface.md:75]: We should reconsider if removing “GraphQL Introspection” term is a good idea as "introspection" is a commonly used term in the GraphQL community and we do have support for it in the schema tab.

Comment (petewitcher-postman) [reply src/pages/docs/sending-requests/graphql/graphql-client-interface.md:75]: Replaced. Thanks Rohan.

**Review by ronork — APPROVED** [review]
LGTM

**Pull request merged**